src/docs/background/off-session-payments/index.md

```diff
@@ -34,7 +34,10 @@ booking period.
 Suppose your sauna rentals marketplace should allow customers to book
 saunas up to a year in advance, but the customer is charged only a
 certain time in advance before the booking. The figure below illustrates
-how a part of your transaction process might look like.
+how a part of your transaction process might look like. The
+[flex-example-processes Github repository](https://github.com/sharetribe/flex-example-processes)
```

Comment:
TODO: Once the example process is in the repo, this link could lead straight to the correct example process